ndex2/client.py

```diff
@@ -318,9 +318,38 @@ def post_multipart(self, route, fields, query_string=None):
 
     def save_new_network(self, cx, visibility=None):
         """
-        Create a new network (cx) on the server
 
-        :param cx: Network cx
+        Create a new network (cx) on the server
+       
+	 .. code-block:: python
+		
+		import ndex2.client
+		my_account="your account"
+		my_password="your password"
+		my_ndex=ndex2.client.Ndex2("http://public.ndexbio.org", my_account, my_password)
+
+		#this method  creates a new network from cx, a python dict in CX format
+
+		# Query the network from my NDEx account, get small subset in cx format
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','INSERT NODE NAME HERE')
```

Comment:
Instead of making the user find a network they can run a neighborhood query on just directly create a network with this fragment of code (you'll have to move the import to the top.

```
from ndex2.nice_cx_network import NiceCXNetwork

my_net = NiceCXNetwork()
my_net.create_node('testnode1')
my_net.set_name('testnetwork')
uri_subset = my_ndex.save_new_network(my_net.to_cx())
```